medicines/doc-index-updater/src/delete_manager/mod.rs

```diff
@@ -405,14 +408,14 @@ mod test {
         let removeable_message = given_we_have_a_delete_message().message;
         let search_client = given_a_search_client_that_cannot_restore_index();
         let storage_client = given_a_storage_client_that_cannot_delete_blob();
-        let storage_container_name = "storage_container_name".to_string();
+        given_the_necessary_env_vars_are_initialised();
```

Comment:
You can only run this test now with `make test` which makes it a bit less flexible as the run test button in vscode won't work for it. 

Is there any way around this? Is it totally necessary to use this env variable in the test, or more accurately, could this be injected into the function under test instead of hard wired to the env variable ?